windows/bootstrap.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/sh
+QTOX_DIR=`pwd`/..
+
+if [ -d $QTOX_DIR/libs ]; then
+    echo "Remove ./libs and redownload/recompile dependencies?"
+    read -p "m/a/N (missing/all/no): " yn
+    yn=$(echo $yn | tr "[:upper:]" "[:lower:]")
+    if [ "$yn" == "a" ]; then
+        rm -rf $QTOX_DIR/libs
+    elif [ "$yn" == "n" ]; then
+        exit -1
+    elif [ "$yn" != "m" ]; then
+        exit -1
+    fi
+fi
+
+mkdir -p $QTOX_DIR/libs
+cd $QTOX_DIR/libs
+
+
+## toxcore
+if [ ! -f "libtoxcore-win32-i686.zip" ]; then
+   wget --no-check-certificate http://jenkins.libtoxcore.so/job/libtoxcore-win32-i686/lastSuccessfulBuild/artifact/libtoxcore-win32-i686.zip
```

Comment:
Indeed but i added this so getting stuff actually works. Weird thing is that browser seems to recognize certificate just fine. In case of wget - should it not use windows certificate store? This is error that it spews anyway:

    C:\Users\Administrator\Desktop\qTox>wget https://jenkins.libtoxcore.so/job/libtoxcore-win32-i686/lastSuccessfulBuild/art
    ifact/libtoxcore-win32-i686.zip
    --2015-01-20 10:40:27--  https://jenkins.libtoxcore.so/job/libtoxcore-win32-i686/lastSuccessfulBuild/artifact/libtoxcore
    -win32-i686.zip
    Resolving jenkins.libtoxcore.so... 104.219.184.40
    Connecting to jenkins.libtoxcore.so|104.219.184.40|:443... connected.
    ERROR: cannot verify jenkins.libtoxcore.so's certificate, issued by `/C=IL/O=StartCom Ltd./OU=Secure Digital Certificate
    Signing/CN=StartCom Class 1 Primary Intermediate Server CA':
    Unable to locally verify the issuer's authority.
    To connect to jenkins.libtoxcore.so insecurely, use `--no-check-certificate'.

Probably most portable thing to do would be bundling root certificates in repo and using *--ca-directory* flag passing the path. But its rather ugly. If you have better ideas im all ears.